### PR TITLE
reboot

### PR DESCRIPTION
**Pull request by charles-uno**
We have quite a bit of existing content. Some good, some less so. And quite a bit of baggage built into the existing organizational structure.

My inclination is to demote the existing content to loose notes, and keep it around for reference. Start the "real" document fresh and iterate. 

Based on the campaign Andy and I tried last winter, I think 5 pages is a better goal than 50.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
 ENV DEBIAN_FRONTEND=noninteractive
 
-RUN apt-get install -y make pandoc
-
-# These packages are pretty big. Let's save a few minutes by putting
-# each in their own layer while we figure things out.
-RUN apt-get install -y texlive-latex-base
-RUN apt-get install -y texlive-fonts-recommended
-RUN apt-get install -y texlive-latex-recommended
+RUN apt-get -y update && apt-get install -y \
```
Comment (agude) [inline Dockerfile:6]: Yeah this is *way* better. Do you want to clear the package cache after as well (same `RUN`)?

**Review by agude — APPROVED** [review]
Looks good! I have one question about the docker file.

**Comment by agude** [comment]
Also (maybe a separate PR) would be nice to have github actions build and post the PDF...

File: Dockerfile (hunk 1 above)
Comment (charles-uno) [reply Dockerfile:6]: Like with `apt-get clean`? I don't think the cache gets stored in the Docker image anyway

**Pull request merged**